app/scripts/controllers/transactions/index.js

```diff
@@ -1218,7 +1275,10 @@ export default class TransactionController extends EventEmitter {
         latestTxMeta,
         'transactions#confirmTransaction - add postTxBalance',
       );
-      this._trackSwapsMetrics(latestTxMeta, approvalTxMeta);
```

Comment:
Do we still need to delete this function?

Reply:
Removed in https://github.com/MetaMask/metamask-extension/pull/21456/commits/86dfac2fc9acf571f3240e2ea59948f8ed6b4a1f